daisy_workflows/image_import/inspection/src/boot_inspect/cli.py

```diff
@@ -51,22 +55,31 @@ def _output_human(results: model.InspectionResults):
   _output_json(results, indent=4)
 
 
-def _inspect_uefi(device):
+def _inspect_boot_loader(device):
   try:
     stream = os.popen('fdisk -l {} -o type'.format(device))
     output = stream.read()
     print(output)
 
+    bios_bootable = False
+    uefi_bootable = False
+    root_fs = ""
+
+    if "BIOS boot" in output:
```

Comment:
String matching the human-readable output of a command is typically error prone and fragile. What alternatives have you investigated? Does libguestfs expose the information that you need? Is there a shared library for fdisk that you can call through a python binding?